lib/atoms/a/template.html

```diff
@@ -1 +1 @@
-<a ng-href="{{$ctrl.href}}" ng-class="$ctrl.class" ng-attr-target="{{$ctrl.target}}">{{$ctrl.title}}</a>
\ No newline at end of file
+<a ng-href="{{$ctrl.lnHref}}" ng-class="$ctrl.lnClass" ng-attr-target="{{$ctrl.lnTarget}}">{{$ctrl.lnTitle}}</a>
```

Comment:
We should use lnContent or something instead of lnTitle, because lnTitle implies the title attribute (sorr,y that was my mistake).